sdk/script/management-sdk.ts

```diff
@@ -90,19 +90,23 @@ class AccountManager {
         });
     }
 
-    public addAccessKey(description: string): Promise<AccessKey> {
-        if (!description) {
-            throw new Error("A description must be specified when adding an access key.");
+    public addAccessKey(friendlyName: string, ttl?: number): Promise<AccessKey> {
```

Comment:
I would prefer that we just call this parameter `name` instead of `friendlyName`, since that is the name we display in the CLI output.